### DISCUSSION TITLE
Port OBS Studio to Apple Silicon ARM64/Aarch64

### DISCUSSION THREAD

**Discussion by DaveFlashNL** [obsproject/obs-studio#5410] [Development Questions] [answered]
### Description
Port OBS Studio to Apple Silicon / Aarch64/ARM64 on Mac.

 ### Motivation and Context
Apple announced their plan to move from intel to Apple Silicon in 2019, and launched their frist devices with the Apple M1 ARM CPU in 2020. More and more manufactures are joining Apple and their move to RISC-based CPU'S. ARM devices are always on, always connected, consume much less energy, and have days of battery life.

### See also
https://github.com/obsproject/obs-studio/wiki/Build-for-Windows-ARM64
and 
https://github.com/tommyvct/obs-studio

for the existing efforts by others to do a windows ARM built properly.

all I ask for is the same for Apple / macOS.

**Comment by WizardCM** [comment, marked as answer]
You can find existing efforts here 

* #5155
* #5063 
* #4714
